### ISSUE TITLE
V5.1.2-beta11 - sudden reboot caused by DNS Tunneling Protection still exist

### ISSUE THREAD

**Issue by shown19** [AdoPiSoft/Beta-Releases#27]
As the title says po, bug parin yong DNS tunneling protection which I experienced from previous versions. I turned it off for now.

**Comment by KoiYaMorph** [comment]
Same po.. at kapag na ON ung DNS Tunneling Protection ung Allow walled garden maboblock or d na ma access..

**Comment by darellep** [comment]
Hello @shown19 @KoiYaMorph 

This has been fixed sa latest beta version. You might want to update your system and recheck this issue and confirm.

Thank you.

**Comment by shown19** [comment]
> Hello @shown19 @KoiYaMorph
> 
> This has been fixed sa latest beta version. You might want to update your system and recheck this issue and confirm.
> 
> Thank you.

Yes, matagal napong nakabeta15 and I no longer experience any sudden reboot caused by DNS tunneling. Thank you.

**Comment by darellep** [comment]
Thank you for the confirmation sir @shown19.
